docs/content.zh/docs/connectors/datastream/kinesis.md

```diff
@@ -593,7 +593,7 @@ KinesisDataStreamsSink<String> kdsSink =
         .setStreamName("your-stream-name")                                        // Required
         .setFailOnError(false)                                                    // Optional
         .setMaxBatchSize(500)                                                     // Optional
-        .setMaxInFlightRequests(16)                                               // Optional
+        .setMaxInFlightRequests(50)                                               // Optional
```

Comment:
How will this impact throttling issues (FLINK-25793)? I assume allowing more parallel requests will increase throttling and reduce throughput? Should we delay this PR until FLINK-25793 is complete?

Reply:
I'm tempted to merge this one now since we would like our value for `maxInFlightRequests` to be 50 eventually. 16 was originally picked arbitrarily I'm afraid, but I feel we will be trying a variety of values for `maxInFlightRequests` during testing throughput/throttling for FLINK-25793. :-)

Reply:
ok